j2k/testData/fileOrElement/switch/caseWithDefaultblock.java

```diff
@@ -0,0 +1,21 @@
+//method
+void foo() {
```

Comment:
should I create a complete java file with main method etc? like  kotlin/j2k/testData/fileOrElement/switch/comlicatedFallDown.java?
I did not do so because there are other .java files in the with similar structure to the one I created, because I followed that structure, like:
 kotlin/j2k/testData/fileOrElement/switch/caseWithBlock.java